src/main/java/nl/windesheim/reporting/builders/SingletonFoundPatternBuilder.java

```diff
@@ -12,25 +13,25 @@
 public class SingletonFoundPatternBuilder extends AbstractFoundPatternBuilder {
 
     /**
-     * Filename of file where pattern is found.
+     * file of file where pattern is found.
      */
-    private final String fileName;
+    private final ClassOrInterface file;
 
     /**
      * Create the builder.
      * @param fileName filename of the file where singleton is found
      */
-    public SingletonFoundPatternBuilder(final String fileName) {
+    public SingletonFoundPatternBuilder(final ClassOrInterface fileName) {
```

Comment:
Parameter is hier nog steeds fileName. Om verwarring te voorkomen deze veranderen naar naam